pkg/receipt/receipt.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package receipt
+
+import (
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"sigs.k8s.io/krew/pkg/index"
+)
+
+func Store(plugin index.Plugin, dest string) error {
```

Comment:
I haven't checked this code path but: is it possible to pass the original manifest (as `[]byte`) here do you think? If it's cleanly/easily possible, we should strive to preserve the verbatim manifest.

It would actually help preserve comments, indentation, and would help preserve default values (or prevent marshaler do defaulting on fields).